src/plugins/IdentityConvIPluginV3/IdentityConvPlugin.h

```diff
@@ -0,0 +1,112 @@
+#ifndef TENSORRT_IDENTITY_CONV_PLUGIN_H
+#define TENSORRT_IDENTITY_CONV_PLUGIN_H
+
+#include <string>
+#include <vector>
+
+#include <cuda_runtime.h>
+
+#include <NvInferRuntime.h>
+#include <NvInferRuntimePlugin.h>
+
+constexpr char const* const kIDENTITY_CONV_PLUGIN_NAME{"IdentityConv"};
```

Comment:
Add comment "Now the plugin name, version, and name space" must be specified for the plugin and plugin creator exactly the same.